common/config/chaintype.go

```diff
@@ -13,22 +13,25 @@ const (
 	ChainArbitrum        ChainType = "arbitrum"
 	ChainMetis           ChainType = "metis"
 	ChainOptimismBedrock ChainType = "optimismBedrock"
-	ChainXDai            ChainType = "xdai"
-	ChainCelo            ChainType = "celo"
-	ChainWeMix           ChainType = "wemix"
-	ChainKroma           ChainType = "kroma"
-	ChainZkSync          ChainType = "zksync"
-	ChainScroll          ChainType = "scroll"
+	// ChainXDai has been renamed to ChainGnosis
+	// Deprecated: use ChainGnosis instead
+	ChainXDai   ChainType = "xdai"
```

Comment:
nit/ a trailing line comment would avoid disrupting the alignment.

Also, pre-existing, but I'm surprised these aren't alpha-sorted. That would help readability.